lib/perl/Genome/InstrumentData/AlignmentResult/Command/PicardRnaSeqMetrics.t

```diff
@@ -3,11 +3,53 @@
 use strict;
 use warnings;
 
+use Test::Exception;
+use Test::MockObject;
+use Test::More tests => 2;
+
 use above 'Genome';
-use Test::More tests => 1;
 
-# This test was auto-generated because './InstrumentData/AlignmentResult/Command/PicardRnaSeqMetrics.pm'
-# had no '.t' file beside it.  Please remove this test if you believe it was
-# created unnecessarily.  This is a bare minimum test that just compiles Perl
-# and the UR class.
-use_ok('Genome::InstrumentData::AlignmentResult::Command::PicardRnaSeqMetrics');
+my $class = 'Genome::InstrumentData::AlignmentResult::Command::PicardRnaSeqMetrics';
+use_ok($class) or die;
+
+subtest 'does_annotation_build_have_required_files' => sub{
+    plan tests => 5;
+
+    my $annotation_build = Test::MockObject->new;
+    $annotation_build->set_always('rRNA_MT_file', 'foo');
+    $annotation_build->set_always('annotation_file', 'foo');
+
+    my $reference_build =  Test::MockObject->new;
+    $reference_build->set_always('id', '1');
+
+    throws_ok(
+        sub{ $class->does_annotation_build_have_required_files(); },
+        qr/but 3 were expected/,
+        'does_annotation_build_have_required_files fails w/o params',
+    );
+    throws_ok(
+        sub{ $class->does_annotation_build_have_required_files($annotation_build, $reference_build); },
+        qr/Cannot proceed\! Missing required files from annotation build: rRNA_MT_file annotation_file/,
+        'does_annotation_build_have_required_files fails when annotation build does not have required files',
+    );
+
+    for my $file_method ( $class->required_files_from_annotation_build ) {
+        $annotation_build->set_always($file_method, $0);
+        throws_ok(
+            sub{ $class->does_annotation_build_have_required_files($annotation_build, $reference_build); },
+            qr/Cannot proceed\! Missing required files from annotation build: /,
+            "does_annotation_build_have_required_files fails when annotation build does not have $file_method",
+        );
+        $annotation_build->set_always($file_method, 'foo');
+    }
+
+    map { $annotation_build->set_always($_, $0) } $class->required_files_from_annotation_build;
+    $annotation_build->set_always('annotation_file', $0);
```

Comment:
This line appears to be redundant to the line above it.

This will fail in this pathological case:
`perl -e 'do "Genome/InstrumentData/AlignmentResult/Command/PicardRnaSeqMetrics.t"'` as `$0` will have the value `-e`.  `__FILE__` might be a better choice.  Another option might be `$^X`. (Or write something in `/tmp`!)